src/avram/callbacks.cr

```diff
@@ -80,6 +104,55 @@ module Avram::Callbacks
     end
   end
 
+  macro after_run(method_name)
+    def after_run(object)
+      {% if @type.methods.map(&.name).includes?(:after_run.id) %}
+        previous_def
+      {% else %}
+        super
+      {% end %}
+
+      {{ method_name.id }}(object)
+    end
+  end
+
+  # Run the given block after the operation runs
+  #
+  # The return value from `run` will be passed to this block.
+  #
+  # ```
+  # class GenerateReport < Avram::Operation
+  #   after_run do |value|
+  #     value == "some report"
+  #   end
+  #
+  #   def run
+  #     "some report"
+  #   end
+  # end
+  # ```
+  macro after_run(&block)
```

Comment:
The `after_save` equivalent doesn't exist yet, but will in a following PR to keep this as small as I can.